api/engineapi/cli.py

```diff
@@ -328,7 +329,8 @@ def add_scores_handler(args: argparse.Namespace) -> None:
                 scores = actions.add_scores(
                     db_session=db_session,
                     leaderboard_id=args.leaderboard_id,
-                    scores=json_input,
+                    scores=[data.Score(**score) for score in json_input],
```

Comment:
Better to prepare variable and return readable in log message that `json.load(f)` not parsed in `data.Score()`